lib/elixir/lib/kernel.ex

```diff
@@ -3271,7 +3271,12 @@ defmodule Kernel do
       if Map.has_key?(fields, :message) do
         @spec message(Exception.t) :: String.t
         def message(exception) do
-          exception.message
+          cond do
+            is_nil(exception.message) ->
```

Comment:
This is wrong. If there the message is missing, it is a bug somewhere. We cannot simply discard it.

Remember: I want my code to fail. I don't want it to silently start discarding wrong arguments or magically figure out a way to make it work based on incomplete information.